04.class/class.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs')
+const { MultiSelect } = require('enquirer')
+const minimist = require('minimist')(process.argv.slice(2))
+
+const reader = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+class Memo {
+  constructor () {
+    this.files = fs.readdirSync('./memo_data', 'utf-8')
+  }
+
+  buildChoice () {
+    return this.files.map((path) => {
+      const arrayOfContents = this.buildArrayOfContents(path)
+      const choice = { name: `${arrayOfContents[0]} in ${path}`, value: path }
+      return choice
+    })
+  }
+
+  buildArrayOfContents (filename) {
```

Comment:
ファイルを開く、ファイルの内容を取ってくる、という役割があるので、buildよりもreadとかfetchみたいな名前の方がベターかもしれません。
あと、ファイルの内容が行ごとに分割された配列をかえすのであれば、こんな名前がいいかもです。

```suggestion
  fetchContentLines (filename) {
```

Reply:
@JunichiIto さんレビューありがとうございます！

変数名が処理の役割を網羅できていないなと感じました。
`arrayOf~`のような値の状態を表しただけの変数名をつけがちなので＋で役割も表せるような変数名をつけていこうと思います。

こちらのコミットで修正しましたのでご確認よろしくお願いします。
[ec4508d](https://github.com/OdenTakashi/js-practices/pull/4/commits/ec4508d4fc6459a24734d162bb268555bf0108c4)